Sources/DictionaryExtensions.swift

```diff
@@ -19,7 +19,7 @@ extension Dictionary {
         var result = self
         dictionaries.forEach { (dictionary) -> Void in
             dictionary.forEach { (key, value) -> Void in
```

Comment:
I did look this over, but, at the top of my head, I couldn't find a better approach. Maybe once the accepted [SE-0165 (Dictionary & Set Enhancements)](https://github.com/apple/swift-evolution/blob/master/proposals/0165-dict.md) is implemented, one could make use of merging initializer to refactor it.